test/cmd.js

```diff
@@ -484,6 +486,143 @@ describe('express(1)', function () {
+          assert.ok(/Usage: express /.test(stdout))
+          assert.ok(/--help/.test(stdout))
+          assert.ok(/--version/.test(stdout))
+          var reg = RegExp('error: option `--es6\' requires Node version ' + MIN_ES6_VERSION)
+          assert.ok(reg.test(stderr))
+          done()
+        })
+      })
+    } else {
+      it('should create basic app', function (done) {
+        run(ctx.dir, ['--es6'], function (err, stdout, warnings) {
+          if (err) return done(err)
+          ctx.files = utils.parseCreatedFiles(stdout, ctx.dir)
+          ctx.stdout = stdout
+          ctx.warnings = warnings
+          assert.strictEqual(ctx.files.length, 16)
+          done()
+        })
+      })
+
+      it('should print jade view warning', function () {
+        assert.ok(ctx.warnings.some(function (warn) {
+          return warn === 'the default view engine will not be jade in future releases\nuse `--view=jade\' or `--help\' for additional options'
+        }))
+      })
+
+      it('should provide debug instructions', function () {
+        assert.ok(/DEBUG=express-1---es6:\* (?:& )?npm start/.test(ctx.stdout))
+      })
+
+      it('should have basic files', function () {
+        assert.notStrictEqual(ctx.files.indexOf('bin/www.js'), -1)
+        assert.notStrictEqual(ctx.files.indexOf('app.js'), -1)
+        assert.notStrictEqual(ctx.files.indexOf('package.json'), -1)
+      })
+
+      it('should have jade templates', function () {
+        assert.notStrictEqual(ctx.files.indexOf('views/error.jade'), -1)
+        assert.notStrictEqual(ctx.files.indexOf('views/index.jade'), -1)
+        assert.notStrictEqual(ctx.files.indexOf('views/layout.jade'), -1)
+      })
+
+      it('should have a package.json file with type "module"', function () {
+        var file = path.resolve(ctx.dir, 'package.json')
+        var contents = fs.readFileSync(file, 'utf8')
+        assert.strictEqual(contents, '{\n' +
+          '  "name": "express-1---es6",\n' +
+          '  "version": "0.0.0",\n' +
+          '  "type": "module",\n' +
+          '  "private": true,\n' +
+          '  "scripts": {\n' +
+          '    "start": "node ./bin/www.js"\n' +
+          '  },\n' +
+          '  "dependencies": {\n' +
+          '    "cookie-parser": "~1.4.5",\n' +
+          '    "debug": "~2.6.9",\n' +
+          '    "express": "~4.17.1",\n' +
+          '    "http-errors": "~1.7.2",\n' +
+          '    "jade": "~1.11.0",\n' +
+          '    "morgan": "~1.10.0"\n' +
+          '  }\n' +
+          '}\n')
+      })
+
+      it('should have installable dependencies', function (done) {
+        this.timeout(NPM_INSTALL_TIMEOUT)
+        npmInstall(ctx.dir, done)
+      })
+
+      it('should export an express app from app.js', function (done) {
+        // Use eval since otherwise early Nodes choke on import reserved word
+        // eslint-disable-next-line no-eval
+        eval(
+          'const { pathToFileURL } = require("node:url");' +
```

Comment:
using require("url") fixes the failing `should export an express app from app.js` test case.

Reply:
I'm confused. When the tests were previously available here in the PR, this test was not failing. On which version is it failing now?

Reply:
Yes, Node v15. I've been focusing on getting the tests to pass for Node 15 just because that's the one that's failing the pipeline, since this change is for node 14 and above.

I don't know why `eval` can't find `node:url`. Would love to know.

Reply:
I'm guessing that 14 works because it was LTS and 15 does not work because it was not LTS. I ran into the same thing between LTS v12, which worked, and non-LTS v13, which did not.

Reply:
Thanks for finding this issue! It's fixed in my latest push.